### PR TITLE
Updated ILO embeddings

### PR DESCRIPTION
**Pull request by dwnoble**
Updated iLO embedings to reflect stat var grouping changes from #4430. Test in explore page with URL params `dc=undata_dev` or `dc=undata_ilo`.

Example: http://localhost:8080/explore#q=unemployment+in+the+usa&client=ui_query&dc=undata_dev

Screenshot 2024-07-03 at 4 13 38 AM

### CODE REVIEW

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 1]
```diff
@@ -1,86 +1,33 @@
...
-dc/topic/ILOEIP2EIPNB,Persons Outside The Labour Force ILO Modelled Estimates
-dc/topic/ILOEIP2JOBNB,"Potential Labour Force And Willing Non-jobseekers ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEIP2JOBRT,"Potential Labour Force And Willing Non-jobseekers Rate ILO Modelled Estimates, as Of Nov. 2023"
-dc/topic/ILOEIP2PLFRT,Potential Labour Force Rate ILO Modelled Estimates
-dc/topic/ILOEIP2WAPRT,"Number of Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Inactivity Rate, ILO Modelled Estimates"
-dc/topic/ILOEIP3EIPNB,Youth Outside The Labour Force
-dc/topic/ILOEIP3WAPRT,"Number of Young Persons of Working Age Outside The Labour Force, as A Percentage of The Working-age Population Youth Inactivity Rate"
+dc/topic/ILOEAR4MMNNB,"Monthly Minimum Wage"
+dc/topic/ILOEARXTLPRT,"Low Pay Rate"
+dc/topic/ILOEESXTMPRT,"Share of Temporary Employees"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:4]: "percentage of.." if that's the intention. "share of" might not result in good match.

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 2]
```diff
@@ -1,86 +1,33 @@
...
 dc/topic/ILOEMP5PIFNB,"Employment Outside The Formal Sector, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5PIFRT,"Share of Employment Outside The Formal Secto, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOEMPNIFLNB,Informal Employment
-dc/topic/ILOEMPNIFLRT,Informal Employment Rate
+dc/topic/ILOEMPNIFLRT,"Informal Employment Rate"
 dc/topic/ILOEMPNORMNB,"Employment, Normative Approach"
-dc/topic/ILOEMPPIFLNB,Employment Outside The Formal Sector
-dc/topic/ILOEMPPIFLRT,Share of Employment Outside The Formal Sector
+dc/topic/ILOEMPPIFLNB,"Employment Outside The Formal Sector"
+dc/topic/ILOEMPPIFLRT,"Share of Employment Outside The Formal Sector"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:16]: same here

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 3]
```diff
@@ -1,86 +1,33 @@
...
-dc/topic/ILOEMP3EMPNB,Youth Employment
-dc/topic/ILOEMP3WAPRT,Youth Employment-to-population Ratio
 dc/topic/ILOEMP5NIFNB,"Informal Employment, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5NIFRT,"Informal Employment Rate, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5PIFNB,"Employment Outside The Formal Sector, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
 dc/topic/ILOEMP5PIFRT,"Share of Employment Outside The Formal Secto, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOEMPNIFLNB,Informal Employment
-dc/topic/ILOEMPNIFLRT,Informal Employment Rate
+dc/topic/ILOEMPNIFLRT,"Informal Employment Rate"
 dc/topic/ILOEMPNORMNB,"Employment, Normative Approach"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:14]: "Employment, Normative Approach" is a bit vague?

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 4]
```diff
@@ -1,86 +1,33 @@
...
-dc/topic/ILOUNE2UNENB,Unemployment ILO Modelled Estimates
-dc/topic/ILOUNE3EAPRT,Youth Unemployment Rate
-dc/topic/ILOUNE3UNENB,Youth Unemployment
-dc/topic/ILOUNE3WAPRT,Youth Unemployment-to-population Ratio
-dc/topic/ILOUNE5EAPRT,"Unemployment Rate, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOUNETUNENB,"Unemployment Labour Force Statistics, LFS"
+dc/topic/ILOGED2LFPRT,"Prime-age Labour Force Participation Rate of Couple Households With Children Under Age 6 ILO Modelled Estimates"
+dc/topic/ILOINJDAYSNB,"Days Lost Due To Cases of Occupational Injury With Temporary Incapacity for Work"
+dc/topic/ILOINJFATLNB,"Cases of Fatal Occupational Injury"
+dc/topic/ILOINJFATLRT,"Fatal Occupational Injuries Per 100'000 Workers"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:21]: 100'000 -> 100000?

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 5]
```diff
@@ -1,86 +1,33 @@
...
+dc/topic/ILOINJNFTLNB,"Cases of Non-fatal Occupational Injury"
+dc/topic/ILOINJNFTLRT,"Non-fatal Occupational Injuries Per 100'000 Workers"
+dc/topic/ILOLAC4HRLNB,"Mean Nominal Hourly Labour Cost Per Employee"
+dc/topic/ILOLAIINSPNB,"Number of Labour Inspectors"
+dc/topic/ILOLAP2LIDRT,"Labour Income Distribution  ILO Modelled Estimates"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:26]: extra space here

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 6]
```diff
@@ -1,86 +1,33 @@
...
+dc/topic/ILOLUUXLU2RT,"Combined Rate of Time-related Underemployment And Unemployment LU2"
+dc/topic/ILOLUUXLUXNB,"Jobs Gap"
+dc/topic/ILOLUUXLUXRT,"Jobs Gap Rate"
+dc/topic/ILOSTRDAYSNB,"Days Not Worked Due To Strikes And Lockouts"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:30]: Number of Days? Give the next one are very similar, this needs to be distinct enough.

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv [hunk 7]
```diff
@@ -1,86 +1,33 @@
...
-dc/topic/ILOUNE3EAPRT,Youth Unemployment Rate
-dc/topic/ILOUNE3UNENB,Youth Unemployment
-dc/topic/ILOUNE3WAPRT,Youth Unemployment-to-population Ratio
-dc/topic/ILOUNE5EAPRT,"Unemployment Rate, According To Standards Adopted At The 19th International Conference of Labour Statisticians ICLS"
-dc/topic/ILOUNETUNENB,"Unemployment Labour Force Statistics, LFS"
+dc/topic/ILOGED2LFPRT,"Prime-age Labour Force Participation Rate of Couple Households With Children Under Age 6 ILO Modelled Estimates"
+dc/topic/ILOINJDAYSNB,"Days Lost Due To Cases of Occupational Injury With Temporary Incapacity for Work"
+dc/topic/ILOINJFATLNB,"Cases of Fatal Occupational Injury"
+dc/topic/ILOINJFATLRT,"Fatal Occupational Injuries Per 100'000 Workers"
+dc/topic/ILOINJNFTLNB,"Cases of Non-fatal Occupational Injury"
```
Comment (shifucun) [inline tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:22]: for this and some other similar ones, if they are count, maybe add "number of xxx"?

**Comment by shifucun** [comment]
the nl test error indicates updating golden files.

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 1 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:4]: Added to alternatives.

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 2 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:16]: Added to alternatives.

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 3 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:14]: Added to alternatives.

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 4 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:21]: Added to alternatives, and gave this feedback to UN

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 5 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:26]: Gave this feedback to UN

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 6 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:30]: Added to alternatives

File: tools/nl/embeddings/input/undata_ilo/sheets_svs.csv (hunk 7 above)
Comment (dwnoble) [reply tools/nl/embeddings/input/undata_ilo/sheets_svs.csv:22]: Added to alternatives

**Comment by dwnoble** [comment]
Thanks @shifucun ! Updated the golden files, and i added all of your suggestions to an "alternatives.csv" file in the ilo nl descriptions directory. I also passed along all of these suggestions to the UN, and once they add them to the stat var descriptions and variable groupings, i'll re-generate these base NL descriptions.

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Pull request merged**